### PR TITLE
Change the default XQUEUE_RABBITMQ_TLS value to false

### LINKED ISSUE #3717: Ficus.1 certs process restarts constantly
/edx/var/log/supervisor/certs-stdout.log shows this over and over:
```
2017-02-27 17:31:43,261 CRITICAL 8266 [openedx_certificates.queue_xqueue] queue_xqueue.py:62 - Unable to get queue length: No JSON object could be decoded
2017-02-27 17:31:44,899 INFO 8271 [requests.packages.urllib3.connectionpool] connectionpool.py:180 - Starting new HTTP connection (1): localhost
```
certs-stderr.log shows:
```
PdfReadWarning: Xref table not zero-indexed. ID numbers for objects will not be corrected. [pdf.py:1503]
Traceback (most recent call last):
  File "/edx/app/certs/certificates/certificate_agent.py", line 197, in <module>
    main()
  File "/edx/app/certs/certificates/certificate_agent.py", line 64, in main
    if manager.get_length() == 0:
  File "/edx/app/certs/certificates/openedx_certificates/queue_xqueue.py", line 57, in get_length
    response = json.loads(request.text)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
```
Changing /edx/app/xqueue/xqueue.env.json from 
```
    "RABBIT_TLS": true,
```
to
```
    "RABBIT_TLS": false,
```
fixes the problem.

Should we default this to false?
XQUEUE_RABBITMQ_TLS is the Ansible variable.

@bdero This was originally your PR: opinions?
@sambapete provided the clue.
@edx/devops other ramifications?

### PR DESCRIPTION
**Pull request by bdero**
This PR changes the default value of `XQUEUE_RABBITMQ_TLS` to be `false`, which makes `xqueue` the same as before #3447 (without TLS).
This is necessary because the RabbitMQ role does not configure it to use TLS by default, so enabling TLS by defauly may break builds like issue #3717 explains.

Along with appending this to the changelog, I also made a fix to the prior related changelogs to make sure the entries match strongly with the Roles involved.

Closes #3717.

CC @nedbat @itsjeyd @pomegranited  

Configuration Pull Request
---

Make sure that the following steps are done before merging

  - [ ] A devops team member has commented with :+1:
  - [x] **(Possibly)** are you adding any new default values that need to be overridden when this goes live?
    - [x] **[DEVOPS-5545](https://openedx.atlassian.net/browse/DEVOPS-5545)** Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
    - [x] Add an entry to the CHANGELOG.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @bdero! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fconfiguration&number=3718"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @bdero! I've created [OSPR-1689](https://openedx.atlassian.net/browse/OSPR-1689) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: CHANGELOG.md [hunk 1]
```diff
@@ -210,14 +210,18 @@
   - Added `MONGO_AUTH` to turn authentication on/off. Auth is now enabled by default, and was previously disabled by default.
 
 - Role: xqueue
-  - Added `EDXAPP_CELERY_BROKER_USE_SSL` to allow configuring celery to use TLS.
+  - Added `XQUEUE_RABBITMQ_TLS` to allow configuring xqueue to use TLS when connecting to the AMQP broker.
 
 - Role: edxapp
   - Added `XQUEUE_RABBITMQ_VHOST` to allow configuring the xqueue RabbitMQ host.
-  - Added `XQUEUE_RABBITMQ_PORT` and `XQUEUE_RABBITMQ_TLS` to allow configuring the RabbitMQ port, and enabling TLS respectively.
+  - Added `XQUEUE_RABBITMQ_PORT` to allow configuring the RabbitMQ port.
```
Comment (itsjeyd) [inline CHANGELOG.md]: @bdero We should move the items in L216-L217 to the `Role: xqueue` entry above.

**Review by itsjeyd — APPROVED** [review]
@bdero This LGTM. I had one comment, :+1: once you address it.

- [x] ~~I tested this:~~ Implements a known fix, cf. https://github.com/edx/configuration/issues/3717#issue-210566286.
- [x] I read through the code
- [x] ~~I checked for accessibility issues~~ N/A
- [x] Includes documentation

File: CHANGELOG.md [hunk 2]
```diff
@@ -210,14 +210,18 @@
...
+  - Added `EDXAPP_CELERY_BROKER_USE_SSL` to allow configuring celery to use TLS.
 
 - Role: ecommerce
   - Added `ECOMMERCE_ENTERPRISE_URL` for the `enterprise` API endpoint exposed by a new service `edx-enterprise` (currently hosted by `LMS`), which defaults to the existing setting `ECOMMERCE_LMS_URL_ROOT`.
 
 - Role: ecommerce
   - Removed `SEGMENT_KEY` which is no longer used.  Segment key is now defined in DB configuration. (https://github.com/edx/ecommerce/pull/1121)
+
+- Role: xqueue
+  - Changed `XQUEUE_RABBITMQ_TLS` default from `true` to `false`.
```
Comment (feanil) [inline CHANGELOG.md]: This should be at the top of the change log, not at the bottom.  New changes should go to the top of the changelog.

**Review by feanil — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (bdero) [reply CHANGELOG.md]: Fixed, thanks.

File: CHANGELOG.md (hunk 2 above)
Comment (bdero) [reply CHANGELOG.md]: Ah ok, fixed.

**Comment by bdero** [comment]
@itsjeyd @feanil Thanks for the reviews - I've addressed your comments!

**Review by feanil — APPROVED** [review]

**Pull request merged**